src/lib/snarky_js_bindings/lib/snarky_js_bindings_lib.ml

```diff
@@ -247,6 +260,24 @@ let optdef_arg_method (type a) class_ (name : string)
   in
   Js.Unsafe.set prototype (Js.string name) meth
 
+(* What's a good name for this? *)
+let arg_optdef_arg_method (type a b) class_ (name : string)
```

Comment:
Does anybody have a better name for this? :D

Reply:
I think it's fine :D